db/services/movie.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import QuerySet
+
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    if not genres_ids and not actors_ids:
+        return Movie.objects.all()
+    elif genres_ids and actors_ids:
+        return Movie.objects.filter(
+            genres__id__in=genres_ids, actors__id__in=actors_ids
+        )
+    elif genres_ids:
+        return Movie.objects.filter(genres__id__in=genres_ids)
+    elif actors_ids:
+        return Movie.objects.filter(actors__id__in=actors_ids)
```

Comment:
You can simplify the get_movies function and make it more concise by using conditional expressions (also known as ternary expressions) and chaining filter calls. Here's a more streamlined version of the function that combines the conditions into a single return statement